addons/mail/views/mail_templates.xml

```diff
@@ -44,6 +44,12 @@
             </xpath>
         </template>
 
+        <template id="assets_common" name="attachment popup assets" inherit_id="web.assets_common">
```

Comment:
Then it should be put in assets_common in addon website_livechat, along with thread.js, not in mail